lib/rspec/matchers/built_in/has.rb

```diff
@@ -90,8 +90,12 @@ def predicate_method_name
           predicate
         end
 
-        def predicate_matches?
-          !!predicate_result
+        def predicate_matches?(value = true)
```

Comment:
Can you remove the space around the `=` here, theres a rubocop rule enforcing no spaces, which I personally disagree with but its failing currently. (Ping @pirj, happy to change the rule but not in this PR).

Reply:
This still needs fixing?

Reply:
Oops, forgot to amend my commit. Done